code/modules/surgery/embalming.dm

```diff
@@ -0,0 +1,31 @@
+/datum/surgery/embalming //Fast and easy way to husk bodys
+	name = "Embalming"
+	desc = "A surgical procedure that prevents a corps from producing."
+	steps = list(/datum/surgery_step/incise,
+				/datum/surgery_step/embalming,
+				/datum/surgery_step/close)
+
+	species = list(/mob/living/carbon/human, /mob/living/carbon/monkey)
+	possible_locs = list(BODY_ZONE_CHEST)
+	requires_bodypart_type = 0
+
+/datum/surgery_step/embalming
+	name = "embalming body"
+	implements = list(/obj/item/hemostat = 100, TOOL_SCREWDRIVER = 35, /obj/item/pen = 15)
+	implements = list(/obj/item/reagent_containers/syringe = 100, /obj/item/pen = 30)
+	time = 10
+	chems_needed = list("drying_agent", "sterilizine")
```

Comment:
why not formaldehyde?

Reply:
atm 15u of the fluid does the same, this is a way for quick and easy round start/gear gotten husking

Reply:
Also could be a way to no clone trait people :honk: